src/gui/widgets/FadeButton.cpp

```diff
@@ -2,7 +2,7 @@
  * FadeButton.cpp - implementation of fade-button
  *
  * Copyright (c) 2005-2009 Tobias Doerffel <tobydox/at/users.sourceforge.net>
- * 
+ *
```

Comment:
Irrelevant to this PR. 
```suggestion
 * 
```